earth_enterprise/src/fusion/autoingest/sysman/khAssetManager.cpp

```diff
@@ -109,6 +109,8 @@ khAssetManager::ApplyPending(void)
 
   notify(NFY_INFO, "Asset cache size = %d", Asset::CacheSize());
   notify(NFY_INFO, "Version cache size = %d", AssetVersion::CacheSize());
+  notify(NFY_INFO, "Asset cache file size = %lu", Asset::CacheObjectSize());
+  notify(NFY_INFO, "Version cache file size = %lu", AssetVersion::CacheObjectSize());
```

Comment:
I feel like the wording here might give the wrong impression. It kind of sounds to me like there's a file on disk that represents each cache, and I believe the cache is only in-memory. I think something like `Asset cache memory footprint` or `Asset cache size in memory` along with units (bytes, presumably?) might be clearer.